data_portal_api.yml

```diff
@@ -0,0 +1,334 @@
+openapi: 3.0.0
+info:
+  version: "1.0.0"
+  title: Copora Data Portal API
+  description: >
+    This is the API for the Copora Data Portal
+servers:
+  - description: production
+    url: https://copora.czi-single-cell.science
```

Comment:
sp: copora -> corpora

Also this will likely be something like `api.corpora.cziscience.com`.